python/examples/contraction/definitions.py

```diff
@@ -17,25 +17,29 @@
 class EinsumProblem(ProblemDefinition):
   """Benchmarking problem definition for einsum.
 
-  Currently only supports two-operand contractions. The textual specification of
-  the contraction to perform is similar to that of np.einsum.
+  Supports one-operand and two-operand einsum specifications. The textual
+  specification of the operation is similar to that of np.einsum.
   """
 
-  def __init__(self, specification: str):
+  def __init__(self, specification: str, flop_count: float):
```

Comment:
flop_count_per_iteration?
Also, this is an int, not a float.

Reply:
sure